lte/gateway/c/session_manager/test/test_set_session_manager_handler.cpp

```diff
@@ -261,11 +274,17 @@ TEST_F(SessionManagerHandlerTest, test_InitSessionContext) {
   std::string session_id = id_gen_.gen_session_id(IMSI1);
   session_enforcer->m5g_init_session_credit(
       session_map, IMSI1, session_id, cfg);
+  EXPECT_EQ(request.mutable_common_context()->ue_ipv4(), "10.20.30.40");
 }
 
 TEST_F(SessionManagerHandlerTest, test_UpdateSessionContext) {
   magma::SetSMSessionContext request;
   set_sm_session_context(&request);
+  request.mutable_rat_specific_context()
+      ->mutable_m5gsm_session_context()
+      ->mutable_pdu_address()
+      ->set_redirect_address_type(magma::RedirectServer::IPV4);
+  request.mutable_common_context()->set_ue_ipv4("10.20.30.40");
```

Comment:
We should pull this out to a const value ""10.20.30.40""

Reply:
Added macros `UE_IPV4` and `UE_IPV6` to `Consts.h` as you suggested